docs/validator/run-val.md

```diff
@@ -30,6 +41,9 @@ echo {your-password to the mining account} > password.txt
 echo {your-password for the bls wallet} > blspassword.txt
 geth --config ./config.toml --datadir ./node --syncmode snap -unlock {the address of your mining account} --password password.txt --blspassword blspassword.txt --mine --vote --allow-insecure-unlock --cache 18000
 ```
+OR
+
+2. Update `node` and `eth.miner` section in `config.toml` file. For more details, please look into this: https://forum.bnbchain.org/t/bnb-smart-chain-testnet-luban-upgrade-announcement/1331 (section: https://forum.bnbchain.org/t/bnb-smart-chain-testnet-luban-upgrade-announcement/1331#h-22important-for-validator-operator-8)
```

Comment:
better not quote the forum here. suggest we should put the sample section here so user don't need to jump back and forth to get the info.

Reply:
Addressed! Added steps from the forum along with the reference.